docs/guides/web-ui/saved-searches.md

```diff
@@ -0,0 +1,89 @@
+# Saved Searches
+
+Logfire's **Saved Searches** feature allows you to quickly access, organize, and share your most useful queries in the web UI. This feature is available on the Live page, and can be tested on the [public demo](https://logfire-eu.pydantic.dev/demo/logfire-demo).
+
+## 1. Opening the Search Panel
+
+To get started, click the **Search & filter with SQL** button to open the search panel.
+
+![Search modal trigger](../../images/guide/browser-saved-searches-trigger.png)
+
+You'll see a panel where you can enter a query using either:
+
+- **Direct SQL**: Write your own SQL `WHERE` clause to filter records.
+- **Natural Language (AI)**: Describe your query in plain English (or another language), and Logfire will generate the SQL for you using AI.
+
+![Search panel](../../images/guide/browser-saved-searches-search-panel.png)
+
+## 2. Writing and Generating Queries
+
+- **Direct SQL**: Type your SQL directly in the editor. Autocomplete and schema hints are available.
+- **Generate SQL with AI**: Click the "Generate SQL with AI" option, then describe what you want (e.g., "all exceptions in the last hour"). The AI will convert your description into a SQL query.
+  - You can also enable the **Include existing SQL code** option. This allows the AI to modify or build upon your current SQL query, rather than starting from scratch.
+
+![Natural language to SQL](../../images/guide/browser-saved-searches-natural-language.png)
+
+- **Filter by...**: Use the filter chips below the editor to quickly add common filters (e.g., tags, service name, level).
```

Comment:
```suggestion
- **Filter by...**: Use the filter pills below the editor to quickly add common filters (e.g., tags, service name, level).
```

nit: I think this is probably the more common name (non-blocking / not a strong preference)